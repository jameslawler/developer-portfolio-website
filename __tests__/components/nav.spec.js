import React from 'react';
import renderer from 'react-test-renderer';

import Nav from '../../components/nav';

describe('With Snapshot Testing', () => {
  it('Nav component should render', () => {
    const result = renderer.create(<Nav />).toJSON();
    expect(result).toMatchSnapshot();
  });
});
