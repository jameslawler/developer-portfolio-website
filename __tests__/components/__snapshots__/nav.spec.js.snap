// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing Nav component should render 1`] = `
<nav
  className="jsx-2055897931"
>
  <ul
    className="jsx-2055897931"
  >
    <li
      className="jsx-2055897931"
    >
      <a
        className="jsx-2055897931"
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        Home
      </a>
    </li>
    <li
      className="jsx-2055897931"
    >
      <a
        className="jsx-2055897931"
        href="https://zeit.co/now"
      >
        ZEIT
      </a>
    </li>
    <li
      className="jsx-2055897931"
    >
      <a
        className="jsx-2055897931"
        href="https://github.com/zeit/next.js"
      >
        GitHub
      </a>
    </li>
  </ul>
</nav>
`;
